### PR TITLE
Add the zone parameter to create_subnet

### PR DESCRIPTION
**Pull request by Dyex719**
Adding the zone parameter to create_subnet

### CODE REVIEW

File: cloudbridge/cloud/interfaces/services.py [hunk 1]
```diff
@@ -740,7 +740,7 @@ def find(self, name, limit=None, marker=None):
         pass
 
     @abstractmethod
-    def create(self, name, network_id, cidr_block, zone=None):
+    def create(self, name, network_id, cidr_block, zone=zone):
```
Comment (nuwang) [inline cloudbridge/cloud/interfaces/services.py]: This should just be a required parameter, with no default value, as follows:
```
def create(self, name, network_id, cidr_block, zone):
```

**Comment by afgane** [comment]
If you rebase your changes on top of this commit https://github.com/gvlproject/cloudbridge/commit/18e3c04f90ccf0c18dcc0ca6ceb228b1efae3722, the tests should pass.

**Comment by Dyex719** [comment]
Does this mean that I have to create a new pull request on top of the new commit that you made? I haven't done this before...

**Comment by afgane** [comment]
You can just merge the `master` branch into your fork. I think the following should take care of it:
```
git remote add upstream https://github.com/gvlproject/cloudbridge
git fetch upstream
git checkout dev
git merge upstream/master
```
That should create a merge commit. Ideally, you could then rebase this for a cleaner history but unless you're eager to learn the process, don't worry about it (and if you do decide to learn, push your changes first because with rebase there's a chance of messing up your local cone).

**Comment by nuwang** [comment]
@Dyex719 Are you planning to do more work on this (see here: https://github.com/gvlproject/cloudbridge/issues/63#issuecomment-339983664) or should we merge in the changes you've made so far?

**Comment by Dyex719** [comment]
Sorry, I'm busy till 15th. I can only work on this after that. Is that okay?

**Comment by nuwang** [comment]
No worries, was just following up on this.

**Pull request merged**